Add a bit more room for rounding errors which cause two tests to fail
on x86_64-musl

--- a/tests/rtest16.mac	2021-10-03 21:31:01.654932345 -0300
+++ b/tests/rtest16.mac	2021-10-03 21:31:51.633283404 -0300
@@ -2229,7 +2229,7 @@
 true;
 
 closeto(li[4](10.0) - (9.6140263862742968515251940747859 - 6.3921313179656069159740055708257*%i),
-  5.618e-15);
+  7.945e-15);
 true;
 
 closeto(li[4](10.0b0) - (9.6140263862742968515251940747859b0 - 6.3921313179656069159740055708257b0*%i),
@@ -2277,7 +2277,7 @@
 true;
 
 closeto(li[5](10.0) - (11.2390407376112991620107110964539 - 3.6796065713019972004384472107791*%i),
-  9.93014e-15);
+  1.25608e-14);
 true;
 
 closeto(li[5](10.0b0) - (11.2390407376112991620107110964539b0 - 3.6796065713019972004384472107791b0*%i),
